client/src/components/Navbar/Navbar.tsx

```diff
@@ -113,82 +123,84 @@ const Navbar: React.FC = () => {
 
   const renderMenuItems = () => {
     // TODO: conditionally render based on profile type
-    return menuItems.map((menu) => {
+    return menuItems.map((menu, index) => {
       if (menu.authenticated) {
-        return loggedInUser && <MenuItem key={menu.resource} {...menu} />;
+        return loggedInUser && <MenuItem key={menu.resource || index} index={index} {...menu} />;
```

Comment:
Please refer to comment above

Reply:
I would argue, that because the list is static and the size of the list will not change, this is not an anti-pattern. This is because we are mapping over the **static** list and returning an equally sized array of either undefined or a react element. Since the position of the element never changes and therefore the index, we are able to use "index" to represent a unique key for each element.

Reply:
@KookiKodes I couldn't see the `menuItems` array within this PR but after having a look at it, I must say that you are right - good point and sorry for the confusion  😄